src/arch/xtensa/configs/alderlake_s_defconfig

```diff
@@ -1,4 +1,5 @@
 CONFIG_TIGERLAKE=y
+CONFIG_FIRMWARE_SHORT_NAME="adls"
```

Comment:
why do we need different firmware if it's the same hardware configuration? I think we need to stop doing this, the mix and match of cpus/chipsets force us to use the chipset configuration and ignore what platform/cpu gets used.